client/device/cdfmc/readsmartlicense.go

```diff
@@ -0,0 +1,25 @@
+package cdfmc
```

Comment:
Please use underscores in the name

Reply:
hmm I think Golang's convention is to use short and concise name with no underscore or mixed caps: https://go.dev/blog/package-names

Reply:
Fair enough. TIL!